omnigibson/examples/objects/draw_bounding_box.py

```diff
@@ -76,7 +76,9 @@ def main(random_selection=False, headless=False, short_exec=False):
         if "3d" not in bbox_modality:
             from omnigibson.utils.deprecated_utils import colorize_bboxes
 
-            colorized_img = colorize_bboxes(bboxes_2d_data=obs[bbox_modality], bboxes_2d_rgb=obs["rgb"], num_channels=4)
+            colorized_img = colorize_bboxes(
+                bboxes_2d_data=obs[bbox_modality], bboxes_2d_rgb=obs["rgb"].cpu().numpy(), num_channels=4
```

Comment:
hmm so we are okay with using numpy for this `colorize_bboxes` function I guess?